pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -28,6 +28,8 @@ const (
 	VirtApiCertSecretName           = "kubevirt-virt-api-certs"
 	VirtControllerCertSecretName    = "kubevirt-controller-certs"
 	CABundleKey                     = "ca-bundle"
+	caPodClusterLocal               = "pod.cluster.local"
```

Comment:
nit: for consistency I'd make these two new constants exported too (i.e., start with a capital letter), like the rest of the constants here.

Reply:
My approach is that constants/variable should never be public without a good reason. It's easier to make things public than make them private in the future.

Reply:
Is consistency a good reason then? :-)
At the general level I agree with your statement, but nevertheless in this specific case we already have quite a bunch of constants already exported, so it won't likely make anything worse. 

Anyway this isn't a big deal either way, so really up to you :-)

Reply:
I get your point :) but I think it's better to do things right even if other things have been done wrong. Also maybe there's a good reason for them being public, I'm not sure why they've been defined that way.

I also found one example for a [private const in there](https://github.com/kubevirt/kubevirt/blob/4ecf0945d82d9b70f0ab4984ff36b5f0e3e4ec0f/pkg/virt-operator/resource/generate/components/deployments.go#L39), just so you don't worry too much about consistency ;)